x-pack/plugins/ingest_manager/server/routes/agent/index.ts

```diff
@@ -42,6 +45,35 @@ import { postNewAgentActionHandlerBuilder } from './actions_handlers';
 import { appContextService } from '../../services';
 import { postAgentsUnenrollHandler } from './unenroll_handler';
 
+const ajv = new Ajv({
+  coerceTypes: true,
+  useDefaults: true,
+  removeAdditional: true,
+  allErrors: false,
+  nullable: true,
+});
+
+function schemaErrorsText(errors: any, dataVar: any) {
+  let text = '';
+  const separator = ', ';
+  for (let i = 0; i < errors.length; i++) {
+    const e = errors[i];
+    text += dataVar + (e.dataPath || '') + ' ' + e.message + separator;
+  }
+  return text.slice(0, -separator.length);
```

Comment:
IIUC this could be also me a `.map` & `.join` like 
```suggestion
  return errors.map((e) => `${dataVar + (e.dataPath || '')} ${e.message}`).join(', ');
```